test/testAdaptors/MorphoBlue/MorphoBlueSupplyAdaptor.t.sol

```diff
@@ -0,0 +1,689 @@
+
+        uint256 assets = 10_000e6;
+        deal(address(USDC), address(this), assets);
+        cellar.deposit(assets, address(this));
+
+        // Figure out how much the whale must borrow to borrow all the loanToken.
+        uint256 totalLoanTokenSupplied = uint256(morphoBlue.market(wethUsdcMarketId).totalSupplyAssets);
+        uint256 totalLoanTokenBorrowed = uint256(morphoBlue.market(wethUsdcMarketId).totalBorrowAssets);
+
+        uint256 assetsToBorrow = totalLoanTokenSupplied > totalLoanTokenBorrowed
+            ? totalLoanTokenSupplied - totalLoanTokenBorrowed
+            : 0;
+        // Supply 2x the value we are trying to borrow in weth market collateral (WETH)
+        uint256 collateralToProvide = priceRouter.getValue(USDC, 2 * assetsToBorrow, WETH);
+
+        deal(address(WETH), whaleBorrower, collateralToProvide);
+
+        vm.startPrank(whaleBorrower);
+        WETH.approve(address(morphoBlue), collateralToProvide);
+
+        MarketParams memory market = morphoBlue.idToMarketParams(wethUsdcMarketId);
+
+        morphoBlue.supplyCollateral(market, collateralToProvide, whaleBorrower, hex"");
+
+        // now borrow
+        morphoBlue.borrow(market, assetsToBorrow, 0, whaleBorrower, whaleBorrower);
+        vm.stopPrank();
+
+        uint256 assetsWithdrawable = cellar.totalAssetsWithdrawable();
+
+        assertEq(assetsWithdrawable, 0, "There should be no assets withdrawable.");
+
+        // Whale repays half of their debt.
+        uint256 sharesToRepay = (morphoBlue.position(wethUsdcMarketId, whaleBorrower).borrowShares) / 2;
+
+        vm.startPrank(whaleBorrower);
+        USDC.approve(address(morphoBlue), assetsToBorrow);
+        morphoBlue.repay(market, 0, sharesToRepay, whaleBorrower, hex"");
+        vm.stopPrank();
+
+        uint256 totalLoanTokenSupplied2 = uint256(morphoBlue.market(wethUsdcMarketId).totalSupplyAssets);
+        uint256 totalLoanTokenBorrowed2 = uint256(morphoBlue.market(wethUsdcMarketId).totalBorrowAssets);
+        uint256 liquidLoanToken2 = totalLoanTokenSupplied2 - totalLoanTokenBorrowed2;
+        assetsWithdrawable = cellar.totalAssetsWithdrawable();
+
+        assertEq(assetsWithdrawable, liquidLoanToken2, "Should be able to withdraw liquid loanToken.");
+
+        // Have user withdraw the loanToken.
+        deal(address(USDC), address(this), 0);
+        cellar.withdraw(liquidLoanToken2, address(this), address(this));
+        assertEq(USDC.balanceOf(address(cellar)), liquidLoanToken2, "User should have received liquid loanToken."); // TODO - Crispy, this should be address(this), but it seems to just be unwinding from morphoBlue and not safetransferring to the test address as it should be doing from morphoBlue (see their codebase for withdraw()).
+    }
+
+    // TODO - Review with Crispy as the pricing is stale although I tried troubleshooting with setMockUpdatedAt.As well, other tests with whale didn't throw stale pricing reverts
```

Comment:
You are updating the timestamp of the mock usdc feed, but I think you also need to update the timestamp of the mock dai feed, cuz the error message is saying DAI pricing is stale, not USDC

Reply:
This has been updated as per our conversation outside of this PR. The test is working now :)